deployer/scripts/install.sh

```diff
@@ -255,7 +255,13 @@ function generate_configmaps() {
     # generate curator configmap
     oc create configmap logging-curator \
       --from-file=config.yaml=conf/curator.yml
-    oc label configmap/logging-curator logging-infra=support # make easier to delete later
+    oc label configmap/logging-curator logging-infra=support
+
+    # generate fluentd configmap
+    oc create configmap logging-fluentd \
+      --from-file=fluent.conf=conf/fluent.conf \
+      --from-file=throttle-config.yaml=conf/fluentd-throttle-config.yaml || return
```

Comment:
@t0ffel - are you suggesting to be able to completely replace fluent.conf and the other hardcoded config files?  or just to be able to provide the configuration we now have to provide with environment variables via a configmap instead?